src/test/java/website/frontrow/util/CollisionHandlerTest.java

```diff
@@ -98,9 +98,9 @@ public void freeAABBTest()
                 Cell.WALL, Cell.EMPTY, Cell.WALL,
                 Cell.WALL, Cell.WALL, Cell.WALL), 3, 3));
         CollisionHandler handler = new CollisionHandler(check);
-        assertFalse(handler.checkLevelAABB(
-                new AABB(new Point(1, 1), new Point(2, 2)), new Point(0, 0)
-        ));
+        Cell cell = handler.checkLevelAABB(
+                new AABB(new Point(1, 1), new Point(2, 2)), new Point(0, 0));
+        assertTrue(cell.getType() == Cell.EMPTY);
```

Comment:
Use assertNotEquals instead of == operator cpmbined with assertTrue, == does not work properly for non-objects.
<testing